### ISSUE TITLE
serverless-offline-kinesis: serverless offline hangins and doesn't start local server

### ISSUE THREAD

**Issue by tylerzey** [CoorpAcademy/serverless-plugins#176]
Hi there!

I'm using serverless-offline and serverless-offline-kinesis as documented. Serverless offline kinesis seems to be hanging upon starting up the local server. 

The local port never comes up. The serverless repo works as expected without the offline-kinesis plugin.

Thanks for any pointers!

Env:
Node: 14

package.json
"serverless": "^2.38.0",
"serverless-offline": "^6.9.0",
"serverless-offline-kinesis": "^4.2.0",

serverless.yml file: 
<img width="521" alt="Screen Shot 2021-04-27 at 9 12 13 PM" src="https://user-images.githubusercontent.com/30305023/116331206-4092d200-a79d-11eb-94f3-08f7daea20c2.png">

**Comment by jhuff078** [comment]
@tylerzey I'm having a similar issue, looks to be when the Kinesis stream has a `-` in the name. Removing that works for me.

**Comment by nicklloyd** [comment]
Same issue, but updating the name has no effect 🙁

**Comment by asamurai** [comment]
same issue

**Comment by tnolet** [comment]
Same issue here. Some debugging shows me that this method hangs, as the Kinesis streams are not created. 

https://github.com/CoorpAcademy/serverless-plugins/blob/6a0e1729b739773a417a268eff23679fb123611d/packages/serverless-offline-kinesis/src/kinesis.js#L47

I saw a bunch other issues addressing and talking about this wait behaviour, but they were all closed.

@godu do you have any insights here.

**Comment by quixote15** [comment]
> @tylerzey I'm having a similar issue, looks to be when the Kinesis stream has a `-` in the name. Removing that works for me.

Removing `-` works for me!

**Comment by ChrisHumboldtEvari** [comment]
I am dealing with this issue at the moment. Changing the name did nothing. Has anyone found an actual solution by any chance?

**Comment by jagomsim** [comment]
Hi all, today I was trying to use this plugin and found this "error". I was able to make it work.

Reviewing the code, I realized that they never really create a stream, and I think that it is supposed to be created beforehand.

What I did to make it work, was:
1. Run a docker image of a kinesis server (e.g., `vsouza/kinesis-local`, `saidsef/aws-kinesis-local`, `ruanbekker/kinesis-local:latest` or directly with a Dockerfile as in https://blog.ruanbekker.com/blog/2019/06/22/play-with-kinesis-data-streams-for-free/).
2. Create a stream (e.g., `aws --endpoint-url http://localhost:4567 kinesis create-stream --stream-name polls`).
3. Run serverless offline as normal
